generators/dockertools/templates/python/Dockerfile-tools

```diff
@@ -11,9 +11,13 @@ RUN apk --update add --virtual build-dependencies libffi-dev openssl-dev python-
   && pip install --upgrade pip \
   && pip install --upgrade -r /tmp/requirements.txt \
   && apk del build-dependencies
-
+<% if (language === "PYTHON") { %>
 ENV FLASK_APP=server
 ENV FLASK_DEBUG=true
+<% }else { %>
+ENV DJANGO_APP=<%= name %>
+ENV DJANGO_DEBUG = true
```

Comment:
Are `DJANGO_DEBUG` and `DJANGO_APP` things? `FLASK_APP` and `FLASK_DEBUG` are necessary because Flask looks at these specific environment variables instead of letting them be passed in as command line arguments (which is pretty annoying imo).

I know Django has a `DEBUG` variable in `settings.py`, so I'm not so sure `DJANGO_DEBUG` exists.